Chat/NBTV.vcxproj

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For some reason i cannot ignore '.vs' folder, but everything else is now ignored correctly.